### ISSUE TITLE
Flip Audience display overlay

### ISSUE THREAD

**Issue by YeveRivkin** [FIRST-Tech-Challenge/scorekeeper#335]
We used the overlay feature in our event yesterday and we placed our cameras in the opposite direction than the scoring system so we needed to flip the cameras in order to see the match correctly with the scoring system. So can you add a feature that flips the blue and red alliances in the video overlay?
[Here ](https://clips.twitch.tv/EvilShyHamburgerOpieOP)is an example of that, try to read the team number on the robot, it's very hard to follow the game when you can't see which team did that.

**Comment by adwiii** [comment]
Closing as duplicate of #306 - I do understand the issue and agree it is a valid point.
